Encrypter.php

```diff
@@ -1,79 +0,0 @@
-<?php
-
-/*
- * This file is part of the Nelmio SecurityBundle.
- *
- * (c) Nelmio <hello@nelm.io>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Nelmio\SecurityBundle;
-
-/**
- * @deprecated EncryptedCookieListener is now deprecated due to high coupling with the deprecated mcrypt extension
- */
-class Encrypter
```

Comment:
It's using the deprecated `mcrypt` extension, but someone could have extended it, if we're releasing this as `v2.11` I think we should keep them.

Reply:
This class has been marked deprecated in 5,5 years ago: [8b3e412be202ec0370ade05059d4515a2d316e84](https://github.com/nelmio/NelmioSecurityBundle/commit/8b3e412be202ec0370ade05059d4515a2d316e84). I think we can safely remove it?

Reply:
It's fine to remove, but it means the next release should become 3.0, and if we do that we may take the chance to remove/cleanup other stuff if needed.